release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -810,6 +810,15 @@ The Node Feature Discovery (NFD) Operator detects hardware features available on
 
 - `cstate status`: If the `intel_idle` driver has C-states or idle states, the `cstate status` label is `true`. Otherwise, it is `false`.
 
+[id="ocp-4-8-poison-pill-operator]
+==== Remediate unhealthy nodes with the Poison Pill Operator
+
+You can use the Poison Pill Operator to allow unhealthy nodes reboot automatically. This minimizes downtime for stateful applications and ReadWriteOnce (RWO) volumes, and restores compute capacity if transient failures occur.
+
+Clusters that do not have  hardware such as the Baseboard Management Controller (BMC) can benefit from using the Poison Pill Operator.
```

Comment:
nit: there's double space between "have" and "hardware"

maybe instead of this sentence, we can make more high-level statement, e.g.:
Poison Pill has no prerequisites and can operate on any type of cluster and hardware.